tasks/transform_conformance/src/test_case.rs

```diff
@@ -268,11 +268,7 @@ impl TestCase for ConformanceTestCase {
                 if let Some(throws) = &babel_options.throws {
                     return throws.to_string();
                 }
-                // The transformation should be equal to input.js If output.js does not exist.
-                let program = Parser::new(&allocator, &input, source_type).parse().program;
-                Codegen::<false>::new("", &input, codegen_options.clone())
-                    .build(&program)
-                    .source_text
+                String::default()
```

Comment:
Great call, having the test always fail is much better than having it pass falsely.